binancial/data/get_klines_realtime.py

```
@@ -1,43 +1,66 @@
+    # Apply nest_asyncio for async compatibility
+    nest_asyncio.apply()
+    
+    # Initialize socket id for proper cleanup
+    socket_id = None
+    
+    def handle_socket_message(msg: dict[str, Any]) -> None:
         try:
-            event_time = dt.datetime.fromtimestamp(msg['E'] / 1000).strftime('%Y-%m-%d %H:%M:%S')
-            kline_start_time = dt.datetime.fromtimestamp(msg['k']['t'] / 1000).strftime('%Y-%m-%d %H:%M:%S')
-            kline_close_time = dt.datetime.fromtimestamp(msg['k']['T'] / 1000).strftime('%Y-%m-%d %H:%M:%S')
+            event_time = datetime.fromtimestamp(msg['E'] / 1000).strftime('%Y-%m-%d %H:%M:%S')
```

Ruff (0.8.2)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)